docs/modules/ROOT/pages/tiered-storage.adoc

```diff
@@ -0,0 +1,42 @@
+= Tiered Storage
+:description: Tiered Storage offers an extension to Hazelcast map, which allows you to store a larger data set than available memory.
+:page-enterprise: true
+:page-beta: true
+
+{description}
+
+See xref:hazelcast:tiered-storage:overview.adoc[Tiered Storage] for more detailed information about the configuration and behavior of Tiered Storage in Hazelcast.
+
+Configuration for Tiered Storage includes setting the local device (local disk) parameters for the Hazelcast CR and providing Tiered Storage details for the Map CR.
+
+== Configuring Local Devices
+
+Native Memory must be enabled for Tiered Storage. You can learn about enabling Native Memory for Hazelcast in xref:native-memory.adoc#example-configuration[Native Memory]
+
+WARNING: Native Memory size(*NativeMemory.Size*) must be at least 1084 MB * Number of Tiered Store Map. For example if you are planning to have 3 Tiered Store Map, you need at least 3 * 1084 MB Native Memory size.
+
+For each given local device configuration, *clusterSize*(3) PVCs will be created and used as local disk.
```

Comment:
You can configure one or more local devices in a single Hazelcast CR. A separate PVC is created for each local device and is used as the local disk for that device. Therefore, the number of created PVCs matches the cluster size for each configured local device. 

For example, if we create two local devices, called device1 and device2, for a Hazelcast CR, called hazelcast, and we have a cluster size of `3`, a total of six PVCs are created as follows:

* device1-hazelcast-0
* device1-hazelcast-1
* device1-hazelcast-2
* device2-hazelcast-0
* device2-hazelcast-1
* device2-hazelcast-2